.github/workflows/test.yml

```diff
@@ -1,12 +1,10 @@
 name: test
 
 on:
-  push:
-    branches:
-      - develop
   pull_request:
     branches:
       - develop
+      - develop-*
```

Comment:
This will need to keep working when the current pipeline is being used with `develop-0.4`